src/context/physical.rs

```diff
@@ -651,6 +651,42 @@ impl SeafowlContext {
                                 .await?;
                             Ok(make_dummy_exec())
                         }
+                        SeafowlExtensionNode::Truncate(Truncate {
+                            table_name, ..
+                        }) => {
+                            let mut table =
+                                self.try_get_delta_table(table_name.clone()).await?;
```

Comment:
Please remove the unnecessary clone of `table_name`, if it works.

```suggestion
                            let mut table =
                                self.try_get_delta_table(table_name).await?;
```